.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mp/game/neo"]
+	path = mp/game/neo
+	url = https://github.com/NeotokyoRebuild/assets
```

Comment:
This isn't going to support people using ssh. Use relative path to support both HTTPS and SSH, see: https://www.damirscorner.com/blog/posts/20210423-ChangingUrlsOfGitSubmodules.html